### DISCUSSION TITLE
Images

### DISCUSSION THREAD

**Discussion by pmig** [glasskube/glasskube#226] [General]
Website images can get uploaded in this thread :-)

**Comment by pmig** [comment]
![Glasskube-CNCF-Landscape](https://github.com/glasskube/glasskube/assets/3041752/54b8adad-e88f-4853-9250-e8c3bbbad993)

**Comment by pmig** [comment]
| Light    | Dark |
| -------- | ------- |
| <img width="1000" alt="01-glasskube-browse" src="https://github.com/glasskube/glasskube/assets/3041752/d3e764e4-7ee0-4ad8-b0eb-d0ebf219160f">  |  <img width="1000" alt="01-glasskube-browse" src="https://github.com/glasskube/glasskube/assets/3041752/323994d6-6b08-4dca-ac59-d29ae6b37f94"> |
|  <img width="1000" alt="02-glasskube-config" src="https://github.com/glasskube/glasskube/assets/3041752/28ceea95-c66d-4f62-8fe4-d4d1be160ad6"> | <img width="1000" alt="02-glasskube-config" src="https://github.com/glasskube/glasskube/assets/3041752/df6bd7d4-7cac-435b-b3a0-31c3cab6069b"> |
| <img width="1000" alt="03-glasskube-deps" src="https://github.com/glasskube/glasskube/assets/3041752/b7d65ac6-de61-4771-b81c-29b7b1926f77"> | <img width="1000" alt="03-glasskube-deps" src="https://github.com/glasskube/glasskube/assets/3041752/9588b3fc-2a87-454e-97ff-b0f7558717bc">  |
| <img width="1000" alt="04-glasskube-reactions" src="https://github.com/glasskube/glasskube/assets/3041752/538419ab-3852-4342-9020-0103da00fc21"> |  <img width="1000" alt="04-glasskube-reactions" src="https://github.com/glasskube/glasskube/assets/3041752/56f08373-fbbe-46fd-820e-fb637114336b"> |
| <img width="1000" alt="05-glasskube-settings" src="https://github.com/glasskube/glasskube/assets/3041752/e8565cf7-6089-4b82-b169-91f5b3ef4c33">|  <img width="1000" alt="05-glasskube-settings" src="https://github.com/glasskube/glasskube/assets/3041752/cf1f1983-78c8-4bb4-9d47-86dbf3a16c4e">  |
| <img width="1000" alt="06-glasskube-updates" src="https://github.com/glasskube/glasskube/assets/3041752/70dd3e59-0a3e-46ee-b193-6e91159ab9d4"> | <img width="1000" alt="06-glasskube-updates" src="https://github.com/glasskube/glasskube/assets/3041752/a6e6dc72-9919-4d15-addf-bc709ec76d9d"> |

**Comment by thekubernaut** [comment]
<img width="1000" alt="05-glasskube-settings" src="https://github.com/glasskube/glasskube/assets/130456438/e2f4472b-5b80-4043-9c78-9ccabd8f3337">
<img width="1000" alt="05-glasskube-settings" src="https://github.com/glasskube/glasskube/assets/130456438/3d8f0017-272f-47b0-b3cd-135c8bc001ad">

**Comment by pmig** [comment]
<img width="1000" alt="glasskube-gitops" src="https://github.com/glasskube/glasskube/assets/3041752/8c359e61-9eec-4413-9c13-bca5cd8710d1">

**Comment by pmig** [comment]
![glasskube](https://github.com/glasskube/glasskube/assets/3041752/54b20ffe-1daf-4905-abc5-37e99e056b02)
